sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/DedicatedHost.cs

```diff
@@ -168,9 +168,12 @@ public override void Validate()
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "Sku");
             }
-            if (PlatformFaultDomain < 0)
+            if (PlatformFaultDomain != null)
```

Comment:
>if (PlatformFaultDomain != null) [](http://example.com/codeflow?start=12&length=32)

nit, consider inverting the if condition and returning early. this reduces indentation

Reply:
this is generated. If needs to be updated, update need to be made in Swager

Reply:
not sure why this file is being updated. does your swagger change have anything related to this?
if not, could you please take this change out of the PR